paasta_tools/setup_tron_namespace.py

```diff
@@ -224,7 +224,7 @@ def main():
             # since we need to print out what failed in either case
             failed.append(service)
 
-    if args.bulk_config_fetch:
+    if args.dry_run and args.bulk_config_fetch:
```

Comment:
I guess we should also skip Tron API calls here in the dry run mode?

Reply:
good catch!